packages/react-vapor/src/components/menu/MenuConnected.tsx

```diff
@@ -18,7 +18,6 @@ export interface IMenuOwnProps {
     buttonSvg?: React.ReactNode;
     customOffset?: number;
     buttonProps?: Partial<IButtonProps>;
-    disabled?: boolean;

```

Comment:
Is this a breaking change?

Reply:
its not, It me last week and I dont think someone used it. Its part of my change for the header in a new story